math/mathcore/src/TMathVectorized.cxx

```diff
@@ -331,7 +331,116 @@ Double_v TMath::LaplaceDistI(Double_v x, Double_t alpha, Double_t beta)
 }
 
 // Double_v TMath::LogNormal(Double_v x, Double_t sigma, Double_t theta=0, Double_t m=1);
-// Double_v TMath::NormQuantile(Double_v p);
+
+Double_v TMath::NormQuantile(Double_v p)
+{
+   // Ensure all input probabilities are valid
+   vecCore::Mask_v<Double_v> invalid_values = (p <= 0) || (p >= 1);
```

Comment:
In these cases, it is probably worth fixing the original code, then adding the vectorized version based on the fixed version.

As for the differences, make sure you don't use fast math in your compilation options. Those can indeed make a difference if the vector and scalar codes are not doing exactly the same thing (floating point operations are not associative, e.g. `a + (b + c)` can be different than `(a + b) + c`, and fast math options turn on unsafe rearrangements of the terms like that).